node/src/GlideClient.ts

```diff
@@ -3,7 +3,12 @@
  */
 
 import * as net from "net";
-import { BaseClient, BaseClientConfiguration, ReturnType } from "./BaseClient";
+import {
+    BaseClient,
```

Comment:
Why do we need stuff which is not PubSub spec? like BaseClient? how did it work before

Reply:
I only added `PubSubMsg` import in this line

Reply:
right